packages/react-components/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -1,4 +1,4 @@
-import { shorthands, mergeClasses, makeStyles } from '@griffel/react';
+import { shorthands, mergeClasses, makeResetStyles, makeStyles } from '@griffel/react';
```

Comment:
_nit:_ Take this opportunity to order these alphabetically

```suggestion
import { makeResetStyles, makeStyles, mergeClasses, shorthands } from '@griffel/react';
```